WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -516,17 +516,30 @@ character_event = {
 	trigger = {
 	    NOT = { year = 1200 }
         NOT = { trait = roman_nostalgia }
+		NOT = {
+			religion = assyrian_pagan
+			religion = assyrian_pagan_reformed
+			religion = egyptian_pagan
+			religion = aten_pagan
+			religion = sanamahi
+			religion = sanamahi_reformed
+			religion = tengri_pagan
+			religion = tengri_pagan_reformed
+			religion = scythian_pagan
+			religion = scythuian_pagan_reformed
+			religion = root_west_pagan
+			religion = root_west_pagan_reformed
+			religion = zun_pagan
+			religion = zun_pagan_reformed
+		}
 		OR = {
-			culture_group = central_germanic
-			culture_group = east_germanic
-			culture_group = west_germanic
-			culture_group = latin
-			culture = romano_british
+			religion_group = christian
+			religion_group = pagan_group
```

Comment:
An improvement. Now, the Roman Nostalgia Event can fire for characters throughout the old Roman lines, as opposed to just one or two rulers. The long list of excluded religions exists to prevent odd occurences like Tengris pining for something they never saw.